apps/_root/app/governance-dashboard/governance/GovernanceBoard.tsx

```diff
@@ -0,0 +1,47 @@
+'use client'
+
+import React, { useState } from 'react'
+
+import { GOV_STATUS, GovernanceItem, GovernanceStatus } from '../lib'
+import { GovernanceItemCard, GovernanceTypeFilter } from '../components'
+
+function isValidGovernanceType(govType: string): govType is GovernanceStatus {
+  return govType in GOV_STATUS
+}
+
+export function GovernanceBoard({
+  governanceItemsMapping,
+}: {
+  governanceItemsMapping: Record<'IMPLEMENTATION' | 'PROPOSAL' | 'DISCUSSION', GovernanceItem[]>
+}) {
+  const [selectedGovType, setSelectedGovType] = useState<GovernanceStatus>('IMPLEMENTATION')
+
+  return (
+    <>
+      <div className="hidden grid-cols-3 gap-12 md:grid">
+        {Object.entries(governanceItemsMapping).map(([key, items]) => (
+          <div key={key} className="space-y-5">
+            <div className="flex items-center gap-2 pl-2.5">
+              <div className={`h-3 w-3 rounded-sm ${isValidGovernanceType(key) ? GOV_STATUS[key].color : ''}`} />
```

Comment:
Dont interpolate dynamics into classNames, this is an anti-pattern. Relevant tailwindcss docs:
![Schermafbeelding 2023-06-22 om 09 25 52](https://github.com/sushiswap/sushiswap/assets/80068137/9c4e4968-935d-46ba-a65a-b6d04ed5d53d)

You can use the classNames() function {classNames('h-3 w-3 rounded-sm',isValidGovernanceType(key) ? GOV_STATUS[key].color : '')}